lib/node_modules/@stdlib/blas/ext/base/dapxsumkbn/include/stdlib/blas/ext/base/dapxsumkbn.h

```diff
@@ -31,7 +31,12 @@ extern "C" {
 /**
 * Adds a scalar constant to each double-precision floating-point strided array element and computes the sum using an improved Kahan–Babuška algorithm.
 */
-double stdlib_strided_dapxsumkbn( const int64_t N, const double alpha, const double *X, const int64_t stride );
+double API_SUFFIX(c_dapxsumkbn)( const CBLAS_INT N, const double alpha, double *X, const CBLAS_INT strideX );
```

Comment:
I just realized that you've been updated the C function names from `stdlib_strided_` to `c_` for APIs in `blas/ext/base`. I *think* this is okay, but it does mean we need to be careful regarding naming collisions with `blas/base`. There, we use the `c_` convention because we essentially have to; CBLAS, an external library, uses that convention, so we need to follow suit because that is what users would expect.

Here, `blas/ext/base/*` are functions unique to stdlib, so we are free to "namespace" accordingly. Without the `stdlib_strided_` qualifier, we run the risk of potential naming collisions when used in a downstream library. The `stdlib_strided_` prefix is a convention we use elsewhere in the project (e.g., `@stdlib/math/strided/*`).

In thinking about this more, I think we should keep the `stdlib_strided_` prefix, rather than changing to `c_` for all packages in `stdlib/blas/ext/*`. Sorry for not catching this sooner.

We can still support `API_SUFFIX`, however, so that does not change.

Reply:
It makes sense to keep the `stdlib_strided` qualifier to avoid the collisions. I'll make the changes.